src/components/List.test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { create } from "react-test-renderer";
+
+import { List, ListTypes } from "./List";
+
+it("renders correctly with all props", () => {
+  const component = create(
+    <List
+      id="test"
+      className="class1 class2"
+      type={ListTypes.Number}
```

Comment:
Can we explicitly test `ListTypes.Button`, too. I want to make sure we use the appropriate tags.

Reply:
Is this meant to be Bullet?

Reply:
Yes. Sorry.